docs/build/tutorials/smart-contract-development/account-abstraction/custom-aa-tutorial.md

```diff
@@ -842,6 +842,12 @@ Make sure you deposit funds on zkSync Era using [one of the available bridges](h
 
 1. In the `deploy` folder, create the file `deploy-factory.ts` and copy/paste the following code, replacing `<WALLET_PRIVATE_KET>` with your private key.
 
+::: warning
```

Comment:
Let's please update all instances where this warning should be added. I believe the other tutorials also need this warning.

Reply:
Done!